app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -412,4 +413,8 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
             }
         }
     }
+
+    private fun rebindListItem(position: Int) {
+        bookmarkView.bookmarkAdapter.notifyItemChanged(position)
```

Comment:
We should avoid this as it breaks the encapsulation. Now the View exposes the adapter which exposes the Interactor (that should be private in the first place)..
All in all according to our architecture Views should only have one method - `update()` that consumes a State - what the user sees, exposing more methods will lead to technical debt.